README.md

```diff
@@ -7,3 +7,158 @@ This repo models a process allowing external guidance authors, approvers and pub
 ### License
 
 This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
+
+## Endpoints
+
+### `POST: /external-guidance/scratch`
```

Comment:
/external-guidance/process/scratch

Reply:
Thanks - I started updating wrong README and copied external-guidance scratch URL :(